bluesky_queueserver/manager/start_manager.py

```diff
@@ -260,38 +274,54 @@ def start_manager():
         config_worker["kafka"]["topic"] = args.kafka_topic
         config_worker["kafka"]["bootstrap"] = args.kafka_server
 
-    if args.profile_collection_path:
-        pc_path = args.profile_collection_path
-        pc_path = os.path.abspath(os.path.expanduser(pc_path))
-        if not os.path.exists(pc_path):
-            logger.error("Profile collection directory '%s' does not exist", pc_path)
-            return 1
-        if not os.path.isdir(pc_path):
-            logger.error("Path to profile collection '%s' is not a directory", pc_path)
+    # Find startup directory
+    if args.profile_name:
+        profile_name = args.profile_name
+        try:
```

Comment:
Prefer

```py
from importlib.util import find_spec

if find_spec("IPython"):
    import IPython
    path_to_ipython = ...
else:
    logger.error(...)
```

See [Slides 175-178](http://www.dabeaz.com/modulepackage/ModulePackage.pdf) for illustration of how `try`/`except` can be confusing here if you are curious.

Reply:
Done.